content/articles/ssl-certificates-email-validation.markdown

```diff
@@ -0,0 +1,163 @@
+
+    ![](/files/dnsimple-ssl-selectapprover.png)
+
+1.  Choose the email address you want to use by clicking on it.
+1.  Click *Send Approver Email* to configure and submit the certificate for validation.
+</div>
+
+If the approver is not in this list or you need time to configure one of those emails, you can also close this page and come back later.
+
+<div class="section-steps" markdown="1">
+##### To select a validation email for a previously purchased certificate
+
+1.  Log into your DNSimple account.
+1.  On the top-nav menu click the <label>Domains</label> tab, locate the relevant domain and click on the name to access the domain page.
+1.  Scroll down to the <label>Active SSL Certificate</label> list and click on the certificate.
+
+    ![](/files/dnsimple-ssl-pagelink-purchased.png)
+
+1.  Click <label>complete the setup process now</label>.
+
+    ![](/files/dnsimple-ssl-completesetup.png)
+
+1.  Read the list of all available approver emails.
+
+    ![](/files/dnsimple-ssl-selectapprover.png)
+
+1.  Choose the email address you want to use by clicking on it.
+1.  Click <label>Send Approver Email</label> to configure and submit the certificate for validation.
+</div>
+
+
+## Select a different email address {#different-email}
+
+If you want to submit your certificate to the Certificate Authority for approval but none of the provided email addresses are working, then you have a few options.
+
+<note>
+Before proceeding, **take a moment to read the information at the beginning of this article to understand the goal of the certificate validation and how it works**. It's important to remember that, as described above in this article, the list of authoritative emails is generated by the Certificate Authority based on the email addresses publicly associated with the domain attached to the certificate.
+</note>
+
+Here's some possible solutions:
+
+1. Temporarily configure one of the email addresses in the list (either as a full mailbox or as an alias/forward to an existing mailbox). If the domain doesn't have any email service associated and you manage the DNS with us, you can use our [email forwarding](/articles/email-forwarding/) service to quickly create an email for `admin@example.com` and forward it to a personal or private email.
+
+1. If the list contains an email address associated with a whois privacy service, temporarily [disable the whois privacy](#whois-privacy) to use the email address from the WHOIS.
+
+1. If you want to use a different email address either from the same domain or a different one, check the [email requirements](#requirements) and update the WHOIS information of the domain to include the email address in at least one of the WHOIS contact (e.g. registrant/owner, technical contact or admin contact).
+
+    <note>
+    Some registries, such as the .IO, do not disclose registrant email therefore it's not possible to select the registrant email for a certificate purchased for one of these TLDs.

```

Comment:
It might be good to add something to this effect in the "Email requirements" section.